Python/specialize.c

```diff
@@ -658,13 +656,13 @@ specialize_dict_access(
         }
         // We found an instance with a __dict__.
         PyObject *value = NULL;
-        Py_ssize_t hint =
-            _PyDict_GetItemHint(dict, name, -1, &value);
-        if (hint != (uint16_t)hint) {
+        Py_ssize_t index =
+            _PyDict_GetItemSpecialize(dict, name, &value);
```

Comment:
You forgot this one

Reply:
Thanks!  I'm glad we have automated tests.